src/col/vct/col/typerules/CoercingRewriter.scala

```diff
@@ -1672,6 +1672,8 @@ abstract class CoercingRewriter[Pre <: Generation]() extends AbstractRewriter[Pr
       case CommunicateX(r,s,t,a) => CommunicateX(r,s,t,a)
       case PVLCommunicate(s, r) => PVLCommunicate(s, r)
       case Communicate(r, s) => Communicate(r, s)
+      case PVLSeqAssign(r, f, v) => PVLSeqAssign(r, f, v)
+      case SeqAssign(r, f, v) => SeqAssign(r, f, v)
```

Comment:
it seems suspicious that there is no typing rule here

Reply:
Ah, yeah